vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -471,13 +494,14 @@ WinRTHttpResource::PerformSendRequest(HttpMethod &&method, Uri &&rtUri, IInspect
       auto inputStream = co_await response.Content().ReadAsInputStreamAsync();
       auto reader = DataReader{inputStream};
 
-      // #9510 - We currently accumulate all incoming request data in 10MB chunks
-      const uint32_t segmentSize = 10 * 1024 * 1024;
+      // Accumulate all incoming request data in 8MB chunks
+      // Note, the minimum apparent valid chunk size is 128 KB
+      // Apple's implementation appears to grab 5-8 KB chunks
+      const uint32_t segmentSize = (reqArgs->IncrementalUpdates ? 128 : 8 * 1024) * 1024;
 
```

Comment:
nit: Be a bit more explicit around scoping:
```suggestion
      const uint32_t segmentSize = (reqArgs->IncrementalUpdates ? 128 : (8 * 1024)) * 1024;
```

otherwise perhaps: